style/modules/carousel.css

```diff
@@ -35,7 +38,7 @@
     user-select: none;
 }
 
-.carousel--dots > * {
+.carousel--dot {
```

Comment:
this is being changed because of how `siemaWithDots` adds the dots (it doesn't put a class on the container, if there even is a container)